chia/data_layer/data_store.py

```diff
@@ -978,7 +979,7 @@ async def insert(
                         await self._insert_ancestor_table(left_hash, right_hash, tree_id, new_generation)
 
         if hint_keys_values is not None:
-            hint_keys_values[bytes(key)] = value
```

Comment:
`key` is `bytes` already. This seems redundant (as long as we can trust mypy. I added the assert above for now, just to make sure we can trust it.